strr-api/devops/vaults.gcp.env

```diff
@@ -5,6 +5,11 @@ AUTH_API_URL="op://API/$APP_ENV/auth-api/AUTH_API_URL"
 AUTH_API_VERSION="op://API/$APP_ENV/auth-api/AUTH_API_VERSION"
 LEGAL_API_URL="op://API/$APP_ENV/legal-api/LEGAL_API_URL"
 LEGAL_API_VERSION_2="op://API/$APP_ENV/legal-api/LEGAL_API_VERSION_2"
+LTSA_API_URL="op://API/$APP_ENV/ltsa/LTSA_API_URL"
+LTSA_API_VERSION="op://API/$APP_ENV/ltsa/LTSA_API_VERSION"
+LTSA_API_AUTH_KEY="op://API/$APP_ENV/ltsa/LTSA_API_AUTH_KEY"
```

Comment:
The key exists. You may change LTSA_API_AUTH_KEY to LTSA_API_KEY .

![image](https://github.com/bcgov/STRR/assets/44277752/1bc835d5-d294-431b-a037-55a3221adadb)